cmd/sink/client.go

```diff
@@ -0,0 +1,20 @@
+package main
```

Comment:
Need license headers
```suggestion
// Copyright KubeArchive Authors
// SPDX-License-Identifier: Apache-2.0

package main
```

Reply:
oops, I missed that. Added it now